webapp/app/partials/main/jobs.html

```diff
@@ -52,13 +52,13 @@
 </div>
 
 <div ng-show="result_sets.length == 0 && !isLoadingRsBatch.appending &&
-              !isLoadingJobs && locationHasSearchParam('revision')"
+              !isLoadingJobs && locationHasSearchParam('revision') && currentRepo.url"
               class="result-set-body unknown-message-body">
   <span ng-init="revision=getSearchParamValue('revision')">
-      <span>Waiting for a push with revision <strong>{{revision}}</strong></span>
-      <a href="{{currentRepo.url}}/pushloghtml?changeset={{revision}}"
-         target="_blank"
-         title="open revision {{revision}} on {{currentRepo.url}}">(view pushlog)</a>
+     <span>Waiting for a push with revision <strong>{{revision}}</strong></span>
+     <a href="{{currentRepo.url}}/pushloghtml?changeset={{revision}}"
+        target="_blank"
+        title="open revision {{revision}} on {{currentRepo.url}}">(view pushlog)</a>
     <span class="fa fa-refresh fa-spin"></span>
```

Comment:
There's some extra indentation here - if you are struggling to get this back to how it was - copy the original lines from the existing file back over your copy, eg:
https://github.com/mozilla/treeherder-ui/blob/master/webapp/app/partials/main/jobs.html#L58